vscode_extension/package.json

```diff
@@ -318,7 +318,7 @@
     "prettier": "^1.19.1",
     "sinon": "^7.5.0",
     "typescript": "^3.7.2",
-    "vsce": "~1.75",
+    "vsce": "~1.76",
```

Comment:
1.76.0 is the first version to allow passing the personal access token via the `VSCE_PAT` variable.